addons/web/static/src/js/model.js

```diff
@@ -429,7 +429,7 @@ odoo.define("web/static/src/js/model.js", function (require) {
          */
         async _notifyComponents() {
             const rev = ++this.rev;
-            const subscriptions = this.subscriptions.update;
+            const subscriptions = this.subscriptions.update ? this.subscriptions.update : [];
```

Comment:
In this case you can use the `or` operator:
```suggestion
            const subscriptions = this.subscriptions.update || [];
```